compiler/natives/src/time/time.go

```diff
@@ -42,9 +42,10 @@ func runtimeNano() int64 {
 	return js.Global.Get("Date").New().Call("getTime").Int64() * int64(Millisecond)
 }
 
-func now() (sec int64, nsec int32) {
+func now() (sec int64, nsec int32, mono int64) {
+	// TODO: Use mono if needed/possible.
 	n := runtimeNano()
-	return n / int64(Second), int32(n % int64(Second))
+	return n / int64(Second), int32(n % int64(Second)), 0
```

Comment:
I'm surprised that returning zero works. Is there a test that actually depends on it?